src/Core/AndroidNative/maui/src/main/java/com/microsoft/maui/glide/fontimagesource/FontModelResourceDecoder.java

```diff
@@ -0,0 +1,60 @@
+package com.microsoft.maui.glide.fontimagesource;
+
+import android.graphics.Bitmap;
+import android.graphics.Canvas;
+import android.graphics.Color;
+import android.graphics.Paint;
+import android.graphics.Typeface;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import com.bumptech.glide.load.Options;
+import com.bumptech.glide.load.ResourceDecoder;
+import com.bumptech.glide.load.engine.Resource;
+import com.bumptech.glide.load.engine.bitmap_recycle.BitmapPool;
+import com.bumptech.glide.load.engine.bitmap_recycle.BitmapPoolAdapter;
+import com.bumptech.glide.load.resource.bitmap.BitmapResource;
+
+public class FontModelResourceDecoder implements ResourceDecoder<FontModel, Bitmap> {
```

Comment:
If classes like this are only used from Java, try removing:
```suggestion
class FontModelResourceDecoder implements ResourceDecoder<FontModel, Bitmap> {
```
And see if that prevents us getting a C# binding for this type.

Reply:
We don't get bindings for these already as I've `<remove-node`'d them.